### DISCUSSION TITLE
Query in ascending or descending order in Mongo DB using a timestamp

### DISCUSSION THREAD

**Discussion by Lishenga** [mikro-orm/mikro-orm#1662] [Q&A]
Is there a way of querying a MongoDB database and get the results ordered in an ascending or descending order using time?
For instance get all users and use the key or "column" createdAt(timestamp) to order the results in an ascending or descending order?

Some like this
```js
const [items, count] = await this.itemRepository.findAndCount({ batch: itemBatch }, { limit, offset, orderBy: { createAt: QueryOrder.ASC } })
```

Any help?

**Comment by B4nan** [comment]
What have you tried and what did not work as expected? Your example looks correct.

**Reply by Lishenga** [reply]
It's working i had a typo, rather than use "createdAt" i was using "createAt" when using the column to order the list
